modules/security-groups/main.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_security_group" "appstream_security_group" {
+  name   = var.appstream_security_group
+  vpc_id = var.appstream_vpc_id
+}
+
+resource "aws_vpc_security_group_ingress_rule" "allow_all_inbbound" {
+  security_group_id = aws_security_group.appstream_security_group.id
+
+  cidr_ipv4   = "10.111.3.0/24"
```

Comment:
Since this is a re-usable module, let's not hard code any IPs, rather let's pass these in as variables.